tests/benchdnn/graph/utils.hpp

```
@@ -92,6 +92,13 @@ enum { CRIT = 0x001, WARN = 0x002, NEED_CLEANUP = 0x004 };
                 (ss)->state = bs.state; \
                 if ((ss)->state == res_state_t::SKIPPED) { \
                     (ss)->reason = bs.reason; \
+                } else if (((ss)->state == res_state_t::UNIMPLEMENTED) \
+                        || ((ss)->state == res_state_t::INVALID_ARGUMENTS)) { \
+                    if (is_nvidia_gpu()) { \
+                        (ss)->state = SKIPPED; \
+                        (ss)->reason = skip_reason::case_not_supported; \
+                        BENCHDNN_PRINT(0, "case unsupported on NV GPU\n"); \
```

May align the format with the general message, such as:
```
BENCHDNN_PRINT(0, "SKIP: the case is unsupported on NV GPU\n"); 
```

---

The condition is wrong in the current way. If it's not Nvidia gpu, the issue will be silently skipped.